plugins/filetrees/nvim-tree.nix

```diff
@@ -41,6 +41,16 @@ in
 
     package = helpers.mkPluginPackageOption "nvim-tree" pkgs.vimPlugins.nvim-tree-lua;
 
+    gitPackage = helpers.mkPackageOption {
+      name = "git";
+      default = pkgs.git;
+    };
```

Comment:
Same?

Reply:
Same as above, just didn't want to deal with git conflicts tbh. I can move them into the corresponding named PR, if you want.